cmake/DownloadGoogleBenchmark.cmake

```diff
@@ -4,8 +4,8 @@ PROJECT(googlebenchmark-download NONE)
 
 INCLUDE(ExternalProject)
 ExternalProject_Add(googlebenchmark
-	URL https://github.com/google/benchmark/archive/v1.2.0.zip
-	URL_HASH SHA256=cc463b28cb3701a35c0855fbcefb75b29068443f1952b64dd5f4f669272e95ea
+	URL https://github.com/google/benchmark/archive/v1.6.1.zip
+	URL_HASH SHA256=367e963b8620080aff8c831e24751852cffd1f74ea40f25d9cc1b667a9dd5e45
```

Comment:
Can you please do it as separate PR? (As it is completely unrelated to change in question, is it?)

Reply:
win-arm64 needs this patch to pass benchmark build:
https://github.com/google/benchmark/commit/ea5a5bbff491fd625c6e3458f6edd680b8bd5452

`benchmark.lib(sysinfo.obj) : error LNK2019: unresolved external symbol __rdtsc referenced in function "__int64 __cdecl benchmark::cycleclock::Now(void)" (?Now@cycleclock@benchmar
k@@YA_JXZ) [C:\kg\python_test\github_packages\cpuinfo\init-bench.vcxproj]
C:\kg\python_test\github_packages\cpuinfo\Debug\init-bench.exe : fatal error LNK1120: 1 unresolved externals [C:\kg\python_test\github_packages\cpuinfo\init-bench.vcxproj]`

So if you still would prefer I can create a new PR with this change, I just wanted to give you the background.